src/main/java/com/devcycle/sdk/server/local/bucketing/LocalBucketing.java

```diff
@@ -61,6 +66,13 @@ private Collection<Extern> setImportsOnLinker() {
         });
         linker.define("env", "abort", Extern.fromFunc(abortFn));
 
+        Func seedFn = WasmFunctions.wrap(store, F64, () -> {
+            Random rand = new Random();
```

Comment:
This has to match the expected impl.
```
 return Date.now() * Math.random();
```

Current unix millis * rand double between 0.0 and 1.0